gui/default/assets/css/overrides.css

```diff
@@ -58,6 +58,7 @@ ul+h5 {
 
 .text-monospace {
     font-family: Menlo, Monaco, Consolas, "Courier New", monospace;
+    font-weight: bold;
```

Comment:
Is `font-weight: bold` really necessary? It seems a little too much in your face…

On a side note, is it really a good idea to add `font-weight: bold` to a class called `.text-monospace`? The class named like that may later be used for other, less important, elements as well, not only ID or API key.

Reply:
At least in my test I still found the ID hard to read with the default weight.  Sure, this isn't quite the right place to add it, mixing up monospace with the font weight.  But we're only using the class for device IDs and the API key, both of which need the same readability.

I don't want to argue for having it in bold, it was purely my subjective perception finding it a little thin with the used font.  We can drop this change if you think it's too heavy.

Reply:
So basically, I think bold looks good when used for the ID in the "Device Identification" modal. However, it seems out of place, at least in my eyes, when used for the ID or the API key in other modals. I think this is because there is a lot of other content in them, while in the "Device Identification" modal, the ID is supposed to be the centre of attention.

How about creating a new CSS class, e.g.

```
.text-important {
    font-weight: bolder;
}
```

and adding it to the ID element in the "Device Identification" modal only? Alternatively, you could use the `style` attribute and add it that way for the single element only.

Reply:
Moved it inside a `<strong>` element instead.

Reply:
Of course 😳. I was overcomplicating things. `<strong>` is also much better in terms of proper HTML semantics.